apps/central-scan/frontend/src/screens/ballot_eject_screen.test.tsx

```diff
@@ -636,3 +640,113 @@ test('shows invalid election screen when appropriate', async () => {
+  });
+  const metadata: BallotMetadata = {
+    ballotStyleId: '1',
+    precinctId: '1',
+    ballotType: BallotType.Precinct,
+    electionHash: 'abcde',
+    isTestMode: false,
+  };
+  fetchMock.getOnce(
+    '/central-scanner/scan/hmpb/review/next-sheet',
+    typedAs<Scan.GetNextReviewSheetResponse>({
+      interpreted: {
+        id: 'mock-sheet-id',
+        front: {
+          image: { url: '/front/url' },
+          interpretation: {
+            type: 'InterpretedHmpbPage',
+            markInfo: {
+              ballotSize: { width: 1, height: 1 },
+              marks: [],
+            },
+            metadata: {
+              ...metadata,
+              pageNumber: 1,
+            },
+            adjudicationInfo: {
+              requiresAdjudication: true,
+              enabledReasonInfos: [
+                {
+                  type: AdjudicationReason.Overvote,
+                  contestId: '1',
+                  optionIds: ['1', '2'],
+                  optionIndexes: [0, 1],
+                  expected: 1,
+                },
+              ],
+              ignoredReasonInfos: [],
+              enabledReasons: [AdjudicationReason.Overvote],
+            },
+            votes: {},
+            layout: {
+              pageSize: { width: 1, height: 1 },
+              metadata: {
+                ...metadata,
+                pageNumber: 1,
+              },
+              contests: [],
+            },
+          },
+        },
+        back: {
+          image: { url: '/back/url' },
+          interpretation: {
+            type: 'InterpretedHmpbPage',
+            markInfo: {
+              ballotSize: { width: 1, height: 1 },
+              marks: [],
+            },
+            metadata: {
+              ...metadata,
+              pageNumber: 2,
+            },
+            adjudicationInfo: {
+              requiresAdjudication: false,
+              enabledReasonInfos: [],
+              ignoredReasonInfos: [],
+              enabledReasons: [AdjudicationReason.Overvote],
+            },
+            votes: {},
+            layout: {
+              pageSize: { width: 1, height: 1 },
+              metadata: {
+                ...metadata,
+                pageNumber: 2,
+              },
+              contests: [],
+            },
+          },
+        },
+      },
+      layouts: {},
+      definitions: {},
+    })
+  );
+
+  const continueScanning = jest.fn();
+  const logger = fakeLogger();
+
+  renderInAppContext(
+    <BallotEjectScreen continueScanning={continueScanning} isTestMode />,
+    { apiClient: mockApiClient, logger }
+  );
+
+  await screen.findByText('Overvote');
+  screen.getByText(
+    'The last scanned ballot was not tabulated because an overvote was detected.'
+  );
+
+  expect(screen.queryByText('Tabulate as is')).not.toBeInTheDocument();
+  expect(screen.queryByText('Remove to adjudicate')).not.toBeInTheDocument();
+
+  userEvent.click(screen.getByText('Ballot has been removed'));
```

Comment:
I noticed this button's text wraps to two lines in your screenshot. I remember working on this screen to try to make sure that didn't happen. I wonder what's going on. We should fix.